Change client's sockets to SSL

NeoteraHub opened the pull request.

NeoteraHub committed:
Changed client args

This update removes passwords from the client

NeoteraHub committed:
Added SSL sockets

NeoteraHub committed:
Merge branch 'main' into client-auth-v2

NeoteraHub committed:
Merge branch 'main' into client-auth-v2

padrezulmiro committed:
feat: Change server socket to SSL socket

NeoteraHub committed:
Merge remote-tracking branch 'origin/server-tls' into client-auth-v2

padrezulmiro committed:
style: Remove dead import

padrezulmiro committed:
fix: Set keystore type used by server to JCEKS

NeoteraHub committed:
Merge remote-tracking branch 'origin/server-tls' into client-tls

NeoteraHub committed:
Set keystore and truststore properties

enderesting committed:
added checks to fix errors when server isn't starting blank

enderesting committed:
Merge branch 'fix-server-reinit' into client-tls

enderesting committed:
path clarity

padrezulmiro committed:
Merge remote-tracking branch 'origin/main' into server-tls

enderesting commented on iotclient/IoTDevice.java:
```diff
@@ -34,24 +34,30 @@ public static void main(String[] args) {
-            userAuth(userid, password);
-            deviceAuth(devid);
-            testDevice();
+            //userAuth(userid, password);
```
Removing this part of code breaks commands, because the server is still coded to receive them at the moment. either leave them in or fix them in a branch

padrezulmiro committed:
fix: Uncomment old authentication process

Brought these functions back to allow the system to still execute
normally, allowing for testing of other, new, features.

padrezulmiro committed:
Merge branch 'server-tls' into client-tls

padrezulmiro committed:
Merge remote-tracking branch 'origin/client-tls' into client-tls
